app/src/common/components/MovieList/MovieElement/ReleaseYear/ReleaseYear.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './ReleaseYear.scss';
+
+import formatYear from '../../../../utils/format-year';
+
+export default class ReleaseYear extends React.Component {
```

Comment:
Similar here. Please, make a functional stateless component from it.

Reply:
done